Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9
+
+RUN mkdir -p /app
+
+COPY . main.py /app/
+
+WORKDIR /app
```

Comment:
This can come immediately after "FROM python:3.9"

Reply:
rearranged as recommended